### PR TITLE
Persist walletconnect connection

### PR DESCRIPTION
**Pull request by hochiw**
- Store walletconnect user information in local storage
- Handle auto reconnect wallet connect
- Moved user balance that is used for display to walletprovider

refs oursky/likedao#272

### CODE REVIEW

File: react-app/src/clients/walletConnectClient.ts [hunk 1]
```diff
@@ -43,26 +44,38 @@ export class WalletConnectWallet extends BaseWallet {
...
+        method: "cosmos_getAccounts",
+        params: [chainInfo.chainId],
+      });
+      window.localStorage.setItem(
+        `${WALLET_CONNECT_PEER_ID}-${connector.peerId}`,
+        JSON.stringify(account)
+      );
+    } else {
+      const serializedAccount = window.localStorage.getItem(
+        `${WALLET_CONNECT_PEER_ID}-${connector.peerId}`
```
Comment (chihimng) [inline react-app/src/clients/walletConnectClient.ts:68]: can we make an util func for obtaining LS key from peer id?

**Review by elise-ng — CHANGES_REQUESTED** [review]

**Comment by hochiw** [comment]
Autoconnect also logs out now, will fix in this PR

**Comment by hochiw** [comment]
> Autoconnect also logs out now, will fix in this PR

Actually it's possible for it to be different accounts even when it's connected automatically, maybe think about this later

**Comment by hochiw** [comment]
@chihimng updated 🙏

**Review by elise-ng — APPROVED** [review]

**Pull request merged**